packages/xo-web/src/common/modal.js

```diff
@@ -381,12 +381,15 @@ export const FormModal = decorate([
         <form id={state.formId}>
           {/* It should be better to use a computed to avoid calling the render function on each render,
             but reaclette(v0.4.0) not allow us to access to the effects from a computed */}
-          {state.component ||
-            (state.render !== undefined &&
-              state.render({
-                onChange: effects.onChange,
-                value: state.value,
-              }))}
+          {state.component !== undefined ? (
+            <state.component onChange={effects.onChange} value={state.value} />
+          ) : (
+            state.render !== undefined &&
+            state.render({
+              onChange: effects.onChange,
+              value: state.value,
+            })
+          )}
```

Comment:
Why this change?

Reply:
This change is a fix. I've followed our practice and I've do it in a separated commit.

Reply:
Is it a fix unrelated to this PR?

Reply:
This fix is needed for this PR, see https://github.com/vatesfr/xen-orchestra/pull/4810/files#diff-5c00d01ffc1fb5facb3a83a9a7688111R60